manifests/r/RStudio/quarto/1.1.165/RStudio.quarto.installer.yaml

```diff
@@ -0,0 +1,38 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: RStudio.quarto
+PackageVersion: 1.1.165
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: uninstallPrevious
+Commands:
+- quarto
+AppsAndFeaturesEntries:
+- DisplayName: Quarto
+  Publisher: Rstudio, PBC
+  ProductCode: '{F72386FC-CBA0-4DE6-A92A-7D0CC6E34625}'
+  InstallerType: wix

```

Comment:
This is [in preparation](https://github.com/microsoft/winget-pkgs/blob/master/FAQ.md#what-should-i-do-if-a-package-is-being-published-by-a-new-publisher) (possibly prematurely) for the RStudio -> Posit name change in October.

Otherwise it is not strictly necessary AFAIK (though I am fuzzy on the ARP details).

Reply:
The rules about the ARP fields have changed as it clogs up the Publishing pipeline if they match the PackageName, Publisher and Version from the locale, so this information should be unique (different version or version in the DisplayName)

Reply:
@OfficialEsco None of these are in fact unique, so it's better to omit them, I suppose? Would a Publisher name change be picked up solely on the `ProductCode` under `Installers` or does the recommendation from the [FAQ](https://github.com/microsoft/winget-pkgs/blob/master/FAQ.md?rgh-link-date=2022-08-31T13%3A43%3A51Z#what-should-i-do-if-a-package-is-being-published-by-a-new-publisher) still stand?
(Probably better to wait for the first version published under the new name though).

Reply:
In this case, yes, it is better to omit them.

The publisher name change from the FAQ really only comes into play when the Package Identifier needs to change because of the publisher change. In this case, the publisher is still RStudio, and so the package Identifier is still `RStudio.quarto`, so no change is necessary

Reply:
Thank you. We'll do the proper change when needed.